docs/topics/practices.rst

```diff
@@ -190,11 +190,10 @@ Dynamic Creation of Item Classes
 ================================
 
 For applications in which the structure of item class is to be determined by
-user input, or other changing conditions, you can dynamically create item
-classes instead of manually coding them.
-
-::
+user input or other changing conditions you can return regular Python 
+dicts from spiders.
```

Comment:
What do you think about making it like `you can return regular Python dicts from spiders since Scrapy 1.0. For older versions, you can dynamically create Item classes::` ?